living_documentation_regime/living_documentation_generator.py

```diff
@@ -78,7 +79,11 @@ def generate(self) -> None:
 
         # Data mine GitHub project's issues
         logger.info("Fetching GitHub project data - started.")
-        project_issues: dict[str, list[ProjectIssue]] = self._fetch_github_project_issues()
+        try:
+            project_issues: dict[str, list[ProjectIssue]] = self._fetch_github_project_issues()
```

Comment:
I disagree with this solution.
I am ok to return False when an empty dict is returned - no sense to continue.
No raise is present in the method `_fetch_github_project_issues.` The exception `LivDocInvalidQueryFormatError` is raised using only one utils method. This method is for validation purposes. What if we call it as part of initial validation? and then be less scared in code?